src/Helper.cs

```diff
@@ -11,13 +11,26 @@ public class Helper
         public const int MIN_SLEEP_MILLISECONDS = 15;
         public const int DEFAULT_VIDEO_FRAME_RATE = 30;
 
+        // Old hardcoded
+        [Obsolete("VP8 in RTP is defined as dynamic type, this may not be used for matching.")]
         public const int VP8_FORMATID = 96;
+        [Obsolete("H264 in RTP is defined as dynamic type, this may not be used for matching.")]
         public const int H264_FORMATID = 100;
 
-        internal static List<VideoFormat> GetSupportedVideoFormats() => new List<VideoFormat>
-        {
-            new VideoFormat(VideoCodecsEnum.VP8, Helper.VP8_FORMATID, VideoFormat.DEFAULT_CLOCK_RATE),
-            new VideoFormat(VideoCodecsEnum.H264, Helper.H264_FORMATID, VideoFormat.DEFAULT_CLOCK_RATE)
-        };
+        private static int _dynFmtIdCounter = VideoFormat.DYNAMIC_ID_MIN;
+
+        internal static List<VideoFormat> GetSupportedVideoFormats() => _supportedVidFormats; // Use predefined list of supported video formats
+
+        private static readonly List<VideoFormat> _supportedVidFormats =
+        [
+            new VideoFormat(VideoCodecsEnum.VP8, _dynFmtIdCounter++, VideoFormat.DEFAULT_CLOCK_RATE),
+            new VideoFormat(VideoCodecsEnum.VP9, _dynFmtIdCounter++, VideoFormat.DEFAULT_CLOCK_RATE),
+            new VideoFormat(VideoCodecsEnum.H264, _dynFmtIdCounter++, VideoFormat.DEFAULT_CLOCK_RATE),
+            new VideoFormat(VideoCodecsEnum.H265, _dynFmtIdCounter++, VideoFormat.DEFAULT_CLOCK_RATE),
+
```

Comment:
When testing this PR H264 negotiations always failed with Chrome. Without a packetisation mode set Chrome will choose mode 0. The sipsorcery lirbary [only supports mode 1](https://github.com/sipsorcery-org/sipsorcery/blob/1df3a79d53aeb70a6132e7381864993ff2a96164/src/net/SDP/SDPAudioVideoMediaFormat.cs#L179).

The fix is to add the packetisation attribute to teh H264 & h265 formats:

```
            new VideoFormat(VideoCodecsEnum.H264, _dynFmtIdCounter++, VideoFormat.DEFAULT_CLOCK_RATE, "packetization-mode=1"),
            new VideoFormat(VideoCodecsEnum.H265, _dynFmtIdCounter++, VideoFormat.DEFAULT_CLOCK_RATE, "packetization-mode=1"),
```